src/machine/machinectl.c

```diff
@@ -1104,6 +1111,9 @@ static int copy_files(int argc, char *argv[], void *userdata) {
         host_path = copy_from ? dest : argv[2];
         container_path = copy_from ? argv[2] : dest;
 
+        if (arg_use_vmspawn)
+                return log_error_errno(SYNTHETIC_ERRNO(EOPNOTSUPP), "Copy %s is only supported for --virt-backend=nspawn", copy_from ? "from" : "to");
```

Comment:
hmm, this shouldn't be refused client-side, but refused server side. And I'd expect they already are, no?

Reply:
Some of them, e.g. restart etc, machined assumes it can do just by sending a signal, so those "work" server side and must be caught client side. I will test what the behaviour is with the other commands.

Reply:
Yep all of the others machined knows only work with container type machines and so it refuses to do them, I'll remove the checks on these methods and leave only the one on restart.

Reply:
Actually cat and edit still need checks, it attempts to create a .nspawn file for it when its asked to edit it.